### ISSUE TITLE
Charmcraft pack on Apple Silicon fails

### ISSUE THREAD

**Issue by VariableDeclared** [canonical/charmcraft#796]
Hi Charmcraft team,

[Steps to reproduce]

1. Install charmcraft via homebrew
2. Install multipass
3. run `charmcraft pack`

The following error is thrown:

```
➜  charm-openidc-test-fixture git:(main) charmcraft pack
Failed to launch VM 'charmcraft-charm-openidc-test-fixture-26152800-0-0-arm64'.res=['arm64'] (may take a while the first time but it's reusable)
* Command that failed: 'multipass launch snapcraft:20.04 --name charmcraft-charm-openidc-test-fixture-26152800-0-0-arm64 --cpus 2 --mem 2G --disk 64G'
* Command exit code: 2
* Command output: b'\r                                                                                \r\r                                                                                \r'
* Command standard error output: b'launch failed: Unable to find an image matching "20.04". Please use `multipass find` for supported remotes and images.\n'
```

Log attached:
[charmcraft-20220628-092203.976852.log](https://github.com/canonical/charmcraft/files/8999126/charmcraft-20220628-092203.976852.log)

[Workaround]

I appreciate this is a bit of an edge case, and the workaround here is to run an x64 VM somewhere and use that, as far as I can tell,

Thanks!

Peter

**Comment by facundobatista** [comment]
Hello! Thanks for taking the time of opening an issue.

Can you please run `multipass version` and `multipass find` and let me know the results?

Thanks!

**Comment by VariableDeclared** [comment]
hello, sure
```
$ multipass version
multipass   1.9.1+mac
multipassd  1.9.1+mac
```
```
multipass find
Image                       Aliases           Version          Description
18.04                       bionic            20220615         Ubuntu 18.04 LTS
20.04                       focal,lts         20220615         Ubuntu 20.04 LTS
21.10                       impish            20220616         Ubuntu 21.10
22.04                       jammy             20220622         Ubuntu 22.04 LTS
anbox-cloud-appliance                         latest           Anbox Cloud Appliance
charm-dev                                     latest           A development and testing environment for charmers
docker                                        latest           A Docker environment with Portainer and related tools
minikube                                      latest           minikube is local Kubernetes
````

**Comment by facundobatista** [comment]
Hello! I finally found the issue. It's multipass not finding the correct image. It is solved in 1.9.2. 

I'm closing this as the issue is not really related to charmcraft, but please test everything works ok after updating multipass and reopen it if something is not behaving correctly.

Thanks!!
